src/tailwind.js

```diff
@@ -8,9 +8,31 @@ module.exports = {
     }),
     // when in doubt, always put in `extend`
     extend: {
+      fontFamily: {
+        roboto: ["Roboto", "sans-serif"],
+        ubuntu: ["Ubuntu", "sans-serif"],
+      },
+      fontSize: {
+        '1.625': '1.625rem',
+        '2.5': '2.5rem',
+      },
+      height: {
+        '120': '30rem',
+        '160': '40rem',
+      },
+      minHeight: {
+        '18': '4.5rem',
+      },
       minWidth: {
         xs: "18rem",
       },
+      maxWidth: {
+        '1.5': '1.5rem',
+      },
+      margin: {
+        '-86': '-21.5rem',
+        '-88': '-22rem',
+      },
```

Comment:
are customised values being reused in 2 or 3 or more places? custom values might end up being forgotten, if they are used only once

if possible, try use near values of what tw provides

Reply:
ok removed most of them and utilizing default values of what tw provides. will continue to remove them.